### ISSUE TITLE
[Feature request] Long distance cables on unloaded chunks

### ISSUE THREAD

**Issue by endervad** [MightyPirates/OpenComputers#3155]
Minecraft version: 1.7.10
OC version: 1.7.4

I'm trying to setup a network between at least 2 players, let's say Player A and Player B. Player A has power generators from IC2 + PC and Player B has a server.
![Scheme](https://user-images.githubusercontent.com/10913278/64719191-66883780-d4d0-11e9-905a-6143135d438e.png)
Ideally we wanted to do it like this:

Player A Power Generators <-> Power Converter <-> Switch <-(OC Cables)-> Capacitor <-> Player B Server

We placed a World Anchor at Player A side so that Power Generators continue to work. But, as it turns out, OC cables need to be permanently loaded as well. [This reply](https://github.com/MightyPirates/OpenComputers/issues/3117#issuecomment-506912104) proves it. IC2 cables don't have this issue, so we could get the server up and running with this scheme:

Player A Power Generators -(IC2 Cables)-> Power Converter -> Player B Server

But we can't have a network because OC cables need to be placed and loaded anyway. We could of course just use quite a bit of World Anchors or any chunk loader plugin, but we'd like to have stock solution. And it also would hurt Minecraft server's performance. This network was planned to be extended to multiple people, but force loading every chunk is too bothersome, especially since cables from other mods don't have this problem.

TL;DR: I'm asking to make OC cables work in unloaded chunks, since we can't have a big network without chunk loaders.

**Comment by payonel** [comment]
i recommend link cards or wireless cards, not cables through unloaded chunks. we would have to be a type of "last known network" and ignore chunk unloads that don't remove components but only the connecting cables. It is a rather major change, and now something we're interested in doing because there are reasonable work arounds (link cards and wireless cards)
